src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -372,4 +388,35 @@ static bool TryFetchException(object payload, out Exception exc)
             return true;
         }
     }
+
+#if NET8_0_OR_GREATER
+    private static object GetErrorType(Exception exc)
```

Comment:
Why choose `object` as the return type?

```suggestion
    private static string GetErrorType(Exception exc)
```